pkg/container-utils/containerutils_test.go

```diff
@@ -20,6 +20,23 @@ import (
 	"testing"
 )
 
+func TestNewContainerRuntimeClient(t *testing.T) {
+	nonExistingSocketPath := filepath.Join(t.TempDir(), "non-existing-socket")
+	for _, runtime := range AvailableRuntimes {
+		rc := RuntimeConfig{
+			Name:       runtime,
+			SocketPath: nonExistingSocketPath,
+		}
+		c, err := NewContainerRuntimeClient(&rc)
+		if err != nil {
+			t.Errorf("Cannot create client for %s: %s", runtime, err)
+		}
+		if c == nil {
+			t.Errorf("Client for %s is nil", runtime)
+		}
```

Comment:
Please is "github.com/stretchr/testify" library:

```suggestion
		require.Nil(t, err)
		require.NotNil(t, c)		
```